torchvision/prototype/transforms/functional/_deprecated.py

```diff
@@ -41,3 +42,12 @@ def rgb_to_grayscale(inpt: Any, num_output_channels: int = 1) -> Any:
     )
 
     return _F.rgb_to_grayscale(inpt, num_output_channels=num_output_channels)
+
+
+def to_tensor(inpt: Any) -> torch.Tensor:
+    # FIXME: should we keep the "if needed" phrase or unconditionally recommend `convert_image_dtype`?
```

Comment:
To get the `to_tensor()` behaviour you need both `to_image_tensor()` and `convert_image_dtype()` calls. We should change the language below to indicate this.